### PR TITLE
refator ipam & layer2, and fix some bug

### PR DESCRIPTION
**Pull request by duanjiong**
remove e2e test from porter

### CODE REVIEW

**Comment by zheng1** [comment]
I think this title is better "refactor IPAM & layer2, remove e2e test and fix some bug".

File: controllers/lb/controller.go [hunk 1]
```diff
@@ -130,36 +118,42 @@ func (r *ServiceReconciler) SetupWithManager(mgr ctrl.Manager) error {
...
 				return ctrl.Result{}, nil
 			}
-			r.Log.Error(t, "Create LB for service failed")
-			return ctrl.Result{RequeueAfter: time.Second * 10}, t
+			return ctrl.Result{}, err
 		}
 	}
-	return ctrl.Result{}, nil
+
+	return ctrl.Result{RequeueAfter: time.Second * 60}, nil
```
Comment (zheng1) [inline controllers/lb/controller.go:156]: How about set `RequeueAfter` with command line parameter

File: doc/layer2.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+kind: KubeProxyConfiguration
+mode: "ipvs"
+ipvs:
+  strictARP: true
+```
+
 # Porter Layer2 Usage
 Porter now support bgp and layer2, and will support non-bgp switch in future. In
-order to distinguish them, file `protocol` was added. 
+order to distinguish them, filed `protocol` was added.
```
Comment (zheng1) [inline doc/layer2.md:16]: To distinguish them, you need to specify `protocol` in the spec

File: controllers/lb/controller.go (hunk 1 above)
Comment (duanjiong) [reply controllers/lb/controller.go:156]: Yes,  many params are hard code before, and should be add to commandline, this will do later.

**Comment by duanjiong** [comment]
> Have you tested these changes manually?

Yes,  e2e test are here, https://github.com/duanjiong/test-framework/tree/master/e2e/network. 
And , i also check it manually.

**Pull request merged**